src/base/SafeCallback.sol

```diff
@@ -1,20 +1,22 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.19;
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
 
 import {IUnlockCallback} from "@uniswap/v4-core/src/interfaces/callback/IUnlockCallback.sol";
 import {IPoolManager} from "@uniswap/v4-core/src/interfaces/IPoolManager.sol";
 import {ImmutableState} from "./ImmutableState.sol";
 
 abstract contract SafeCallback is ImmutableState, IUnlockCallback {
-    error NotManager();
+    error NotPoolManager();
 
-    modifier onlyByManager() {
-        if (msg.sender != address(manager)) revert NotManager();
+    constructor(IPoolManager poolManager) ImmutableState(poolManager) {}
```

Comment:
yeah same comment here.. in core we always reference it as `manager` which I honestly like. Why the change?

Reply:
Changed it because I think `manager` is vague in periphery where we have the concept of both a "position manager" and a "pool manager"